File: integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/exceptions/MaxFileSizeExceededException.java
package no.difi.meldingsutveksling.exceptions;

import org.springframework.http.HttpStatus;

public class MaxFileSizeExceededException extends HttpStatusCodeException {
    public MaxFileSizeExceededException(String size, String service, String limit) {
        super(HttpStatus.BAD_REQUEST, MaxFileSizeExceededException.class.getName(), size, service, limit);
    }
}

File: integrasjonspunkt/src/main/kotlin/no/difi/meldingsutveksling/exceptions/MaxFileSizeExceededException.kt
package no.difi.meldingsutveksling.exceptions

import org.springframework.http.HttpStatus

class MaxFileSizeExceededException(size: String, service: String, limit: String): HttpStatusCodeException(HttpStatus.BAD_REQUEST, MaxFileSizeExceededException::class.qualifiedName, size, service, limit)